src/csharp/experimental/build_native_ext_for_ios.sh

```diff
@@ -28,7 +28,7 @@ function build {
     PATH_CC="$(xcrun --sdk $SDK --find clang)"
     PATH_CXX="$(xcrun --sdk $SDK --find clang++)"
 
-    CPPFLAGS="-O2 -Wframe-larger-than=16384 -arch $ARCH -isysroot $(xcrun --sdk $SDK --show-sdk-path) -mios-version-min=9.0 -DPB_NO_PACKED_STRUCTS=1"
```

Comment:
I'm not convinced that this is actually enough to put the boringssl symbol prefixes in place.
Have you tested this or was this just an experiment?

https://github.com/google/boringssl/blob/master/BUILDING.md#building-with-prefixed-symbols

Reply:
I'm using it in our own projects and works like a charm. I checked the symbols using `nm` and I double confirmed it once included in the project where grpc and openssl were clashing, and they are no more. So yes, that's all needed to get it up and running :) Let me know if I can help you further.

Reply:
To explain it a little bit:
- `BORINGSSL_PREFIX` will kick in this logic https://github.com/google/boringssl/blob/1cf78cd290542e8575c5139907d61dbad5b6f49d/include/openssl/base.h#L79
- Luckily, in the repo this file is already provided ( and maintained through your support script ) in this location: https://github.com/grpc/grpc/tree/master/src/boringssl
- Although as it's a global header, all we need to do is make sure the compiler can resolve it, so we include that path in the global search paths by using `-Isrc/boringssl`

All the rest will be auto-wired up by boringssl and when grpc will statically link to it, will find symbols already renamed.